samples/06.using-cards/src/main/java/com/microsoft/bot/sample/usingcards/Cards.java

```diff
@@ -73,29 +73,24 @@ public static ThumbnailCard getThumbnailCard() {
     public static ReceiptCard getReceiptCard() {
         ReceiptCard receiptCard = new ReceiptCard();
         receiptCard.setTitle("John Doe");
-        receiptCard.setFacts(
-            new Fact("Order Number", "1234"), new Fact("Payment Method", "VISA 5555-****"));
-        receiptCard.setItems(
-            new ReceiptItem(){{
-                setTitle("Data Transfer");
-                setPrice("$ 38.45");
-                setQuantity("368");
-                setImage(new CardImage("https://github.com/amido/azure-vector-icons/raw/master/renders/traffic-manager.png"));
-            }},
-            new ReceiptItem() {{
-                setTitle("App Service");
-                setPrice("$ 45.00");
-                setQuantity("720");
-                setImage(new CardImage("https://github.com/amido/azure-vector-icons/raw/master/renders/cloud-service.png"));
-            }}
-            );
+        ReceiptItem receiptItem1 = new ReceiptItem();
+        receiptItem1.setTitle("Data Transfer");
+        receiptItem1.setPrice("$ 38.45");
+        receiptItem1.setQuantity("368");
+        receiptItem1.setImage(new CardImage("https://github.com/amido/azure-vector-icons/raw/master/renders/traffic-manager.png"));
+        ReceiptItem receiptItem2 = new ReceiptItem();
+        receiptItem2.setTitle("App Service");
+        receiptItem2.setPrice("$ 45.00");
+        receiptItem2.setQuantity("720");
+        receiptItem2.setImage(new CardImage("https://github.com/amido/azure-vector-icons/raw/master/renders/cloud-service.png"));
+        receiptCard.setFacts(new Fact("Order Number", "1234"), new Fact("Payment Method", "VISA 5555-****"));
+        receiptCard.setItems(receiptItem1, receiptItem2);
```

Comment:
I guess that it will be better to rename the `receiptItem1` and `receiptItem2` as follows:
- `receiptItem1` -> `receiptDataTransfer`
- `receiptItem2` -> `receiptAppService`